src/services/WizardService.ts

```diff
@@ -8,12 +9,18 @@ export interface IWizardState {
   registrationData: IRegistrationData;
 }
 
-export interface IWizardStage {
+export interface IWizardStage<Errors = any> {
```

Comment:
Is it possible to typehint all of these additional `any`s? I can help with that. Maybe `Errors` can be an `object`?